### PR TITLE
Export constant variables

### PR DESCRIPTION
**Pull request by dskkato**
`jp2k` crate uses `OPJ_DPARAMETERS_DUMP_FLAG` constant as a flag in the decoding step, but the current `openjpeg-sys` seems to stop exporting it, which breaks the jp2k crate (when used with this crate as ffi).

https://github.com/kardeiz/jp2k/blob/8a5dfc26a25bd1e1c2efd9cb0c263a954a7f4531/src/lib.rs#L284

There are log of diffs in this patch, but most of them came from comment block, and `#[derive(Debug, Copy, Clone)]` attributes (which becomes not to be generated). So I think we can safely update.

### CODE REVIEW

**Comment by kornelski** [comment]
Thanks for the PR.

Besides changes to the constants, this PR undoes code formatting in doc comments. Can you avoid that?

**Comment by dskkato** [comment]
I cancelled the code formatting part. Could you please check again?

File: src/ffi.rs [hunk 1]
```diff
@@ -804,15 +857,15 @@ extern "C" {
...
     ///
-    /// * `image` —          image to be destroyed
+    /// * 'image' —          image to be destroyed
     pub fn opj_image_destroy(image: *mut opj_image_t);
 }
 extern "C" {
     /// Creates an image without allocating memory for the image (used in the new version of the library).
     ///
-    /// * `numcmpts` —     the number of components
-    /// * `cmptparms` —    the components parameters
```
Comment (kornelski) [inline src/ffi.rs:814]: I mean the change from `` ` `` to `'` in these doccomments

Comment (dskkato) [reply src/ffi.rs:814]: Hm...I didn't do any format, so I think this change is due to bindgen version difference.

Comment (kornelski) [reply src/ffi.rs:814]: There are more differences - e.g. `Copy` bounds have been removed from a few structs. That's a breaking change.

**Comment by kornelski** [comment]
I've cherry-picked the constants. Thanks.

**Comment by dskkato** [comment]
I am not sure which version of bindgen was used, so I am sorry for the diffs such as no longer derive Copy. If it is possible, could you please generate the code in your environment if I send you a patch only for the script?

**Comment by dskkato** [comment]
Thanks!

**Pull request closed without merging**